Examples/Layouts/README.md

```diff
@@ -7,3 +7,9 @@ Please refer the Provisioning --> Desktop Layout section in Cisco Webex Contact
 * **Header_Added_Desktop Layout.json** - This layout have RSS widget added in Header Section
 * **Demo_Widgets_Added_Desktop Layout.json** - This layout have RSS widget in Header, map widget and covid-by-location widget in the desktop layout (Note: google map API key is required for these widget in attribute section)
 
+**Disclaimer**
+
+These layouts are only samples and are NOT guaranteed to be bug free and production quality.
```

Comment:
hmm... thy are bug free thought :) and production quality. this makes us sound very unprofessional. It;s like we don't know what we're doing. I would rather say: 
"These layouts are for reference only, and might require additional configuration to function properly (e.g. generating a unique API key for Google maps, etc). Please note, any Cisco hosted generic demo widgets are for demo purposes only and will require you to obtain an independent hosting solution if you choose to consume sample widgets as is."

Reply:
They may be bug free today but if something changes in future how we are going to support it ?
For google map api I have already added a comment and I can add another comment for covid-by-location widget also.
We are not saying its they are buggy but we do not gaurantee them to be bug free. That was my thought :)

Reply:
Right. I would rather add something to an extent: "In case you decide to use Example widgets as is and discover an issue, please feel free to file it at https://github.com/CiscoDevNet/webex-contact-center-widget-starter/issues

Reply:
Let me know if this is correct else give me the final line to be added .

These layouts are for reference only and some of these might required additional configuration (e.g API key for Google Maps) to function properly. Please note, all hosted images and widgets are for demo purposes only and it is always recommended to obtain an independent hosting solution for these. If any issue is discovered in Example widgets please feel free to raise it at **https://github.com/CiscoDevNet/webex-contact-center-widget-starter/issues**

Reply:
yes i love this!

Reply:
Done :)